cmd/utils/dev.go

```diff
@@ -246,13 +246,16 @@ func GetDownCommand(devPath string) string {
 	return okDownCommandHint
 }
 
-func GetApp(ctx context.Context, dev *model.Dev, c kubernetes.Interface) (apps.App, bool, error) {
+func GetApp(ctx context.Context, dev *model.Dev, c kubernetes.Interface, isRetry bool) (apps.App, bool, error) {
 	app, err := apps.Get(ctx, dev, dev.Namespace, c)
 	if err != nil {
 		if !errors.IsNotFound(err) {
 			return nil, false, err
 		}
 		if dev.Autocreate {
+			if isRetry && isAppDown(ctx, dev, c) {
```

Comment:
why do we need to check this here?

Reply:
That checks that we didn't lose the connection to the development environment.
For example if an user deletes the pod we have to recreate we have the deployment up but if an user runs `okteto down` we don't have the deployment up so we check that we don't have to recreate the deployment

Reply:
should it be `doesntexist` then? instead of `appdown`?